generated/nifake/nifake.proto

```diff
@@ -138,10 +175,10 @@ enum MobileOSNames {
 
 enum NiFakeInt32AttributeValues {
   NIFAKE_INT32_UNSPECIFIED = 0;
```

Comment:
This one is unexpected (to me) since there's `GrpcColorOverride.GRPC_COLOR_OVERRIDE_UNSPECIFIED` above.

Reply:
So the "AttributeValues" enums are sort of special, they're populated for an attribute type from other enum values. But for UNSPECIFIED (0) enum value for the AttributeValue enums it'll just populate one with the `NIFAKE_INT32_UNSPECIFIED = 0;` instead of multiple UNSPECIFIED's from each enum. This might not be the best example since its just made up of one enum (GrpcColorOverride), but if you look at switch's proto for [NiSwitchInt32AttributeValues ](https://github.com/ni/grpc-device/blob/38d7abec10b2a79ec34c2595fa08f09b5c102afa/generated/niswitch/niswitch.proto#L154) you can see it just has one UNSPECIFIED instead of grabbing them from all the different enums that are used to populate it.

Anyway, this is unchanged behavior and you can see that this same sort of scheme also applies to `NiFakeReal64AttributeValuesMapped` where it just populates a `NIFAKE_REAL64_MAPPED_UNSPECIFIED = 0` and doesn't grab the name from `FLOAT_ENUM_UNSPECIFIED`.